multiqc_docs/sidebar.js

```diff
@@ -50,8 +59,8 @@ export default {
       },
       {
         "type": "category",
-        "label": "Supported tools",
-        "collapsed": true,
+        "label": "Modules",
```

Comment:
I've been trying to get away from the term "Modules" as it means so many different things. With the unified search it'll also come up for nextflow modules, nf-core modules, etc etc.

Is there a reason to move away from "Supported Tools"?

Reply:
We call them "Modules" elsewhere in the docs and the codebase, e.g. "Writing new modules" in the development section.
<img width="269" alt="Screenshot 2024-10-05 at 12 38 55" src="https://github.com/user-attachments/assets/357f1f78-587e-489c-9387-117e12491423">
So I'd say if we want to move away, we would be moving away from "Modules" :) Happy to give it a shot at least across the modules. E.g. "Writing new modules" -> "Adding tools", etc.

Reply:
Yeah, maybe you're right and we should move away entirely.

I'd at least like to leave this page as "Supported Tools" anyway. The folks looking for these pages are typically end users and not developers. I think "Supported Tools" is more user friendly and intuitive.